### PR TITLE
Add import documentation for cronjob

### PR DESCRIPTION
**Pull request by pierlucg-xs**
### Description

Add import documentation for CronJob.

### Release Note
Release note for [CHANGELOG](https://github.com/hashicorp/terraform-provider-kubernetes/blob/main/CHANGELOG.md):

```release-note
NONE
```

### References

Fixes https://github.com/hashicorp/terraform-provider-kubernetes/issues/557

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### CODE REVIEW

**Review by dak1n1 — APPROVED** [review]
Looks good. Thanks for this contribution!

**Comment by dak1n1** [comment]
@pierlucg-xs  It looks like the CLA bot is waiting for your signature before this can be merged. Could you please sign the contributor license agreement? (I think it just gives us permission to use your contribution). Thanks!  https://cla.hashicorp.com/hashicorp/terraform-provider-kubernetes?pullRequest=1237

**Comment by iBrandyJackson** [comment]
@pierlucg-xs - hi, and thanks again for this contribution! 

Please sign the CLA and we will work to get this merged, otherwise, we will be closing this PR due to inactivity in the next 30 days. This helps our maintainers find and focus on the active issues. Thank you.

**Pull request closed without merging**
